docker/owlbot/java/tests/new-version/golden/google-cloud-foo-bom/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-foo-bom</artifactId>
+  <version>0.1.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-foo:current} -->
```

Comment:
Is this the value that would have been incorrect.

Reply:
Yeah, you can see the isolated failing check on the merge commit.

Reply:
From the log:
```
--- input/proto-google-cloud-foo-v2/pom.xml
+++ golden/proto-google-cloud-foo-v2/pom.xml
@@ -10,7 +10,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-foo-parent</artifactId>
-    <version>0.0.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-foo:current} -->
+    <version>0.1.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-foo:current} -->
   </parent>
   <dependencies>
     <dependency>
```